RELEASES.md

```diff
@@ -77,18 +77,30 @@ Support horizons will be defined corresponding to a release branch, identified
 by `<major>.<minor>`. Releases branches will be in one of several states:
 
 - __*Next*__: The next planned release branch.
-- __*Active*__: The release branch is currently supported and accepting patches.
+- __*Active*__: The release is a stable branch which is currently supported and accepting patches.
 - __*Extended*__: The release branch is only accepting security patches.
+- __*LTS*__: The release is a long term stable branch which is currently supported and accepting patches.
 - __*End of Life*__: The release branch is no longer supported and no new patches will be accepted.
 
-Releases will be supported up to one year after a _minor_ release. This means that
+Releases will be supported at least one year after a _minor_ release. This means that
 we will accept bug reports and backports to release branches until the end of
 life date. If no new _minor_ release has been made, that release will be
 considered supported until 6 months after the next _minor_ is released or one year,
 whichever is longer. Additionally, releases may have an extended security support
 period after the end of the active period to accept security backports. This
 timeframe will be decided by maintainers before the end of the active status.
 
+Long term stable (_LTS_) releases will be supported for at least three years after
+their initial _minor_ release. These branches will accept bug reports and
+backports until the end of life date. They may also accept a wider range of
+patches than non-_LTS_ releases to support the longer term maintainability of the
+branch, including library dependency, toolchain (including Go) and other version updates
```

Comment:
This doesn’t seem different from non-LTS

Reply:
We do update Go, certainly for patch releases but normally only major release when a version of Go is going out of support. Its not much different, just calling it out explicitly and we likely want to keep the Go version closer to the latest rather than in non-lts only updating when we have to.